lib/wallet/wallet.js

```diff
@@ -1195,12 +1197,14 @@ class Wallet extends EventEmitter {
     if (rate == null)
       rate = await this.wdb.estimateFee(options.blocks);
 
-    let coins;
```

Comment:
I think there should be a typecheck on `options.coins` to make sure that it is an array

Reply:
Assertion added